posts/2020/11/Design In Open Source

```diff
@@ -0,0 +1,187 @@
+and energy from design itself.
+
+### Community Trust and Recognition
+
+**TG: ** A developer decides to contribute to their favorite open source project for the first  
+time. On the PR, there’s one person who keeps asking them lots of questions, attaching a lot of  
+images, and requesting changes. They sound like they understand the project, but they don’t have  
+the contributor tag and there isn’t any record of them working directly with the project. In fact,  
+there’s barely anything at all on their Github profile. Who could it be? Probably a designer.
+
+**IPF: ** In my experience, communities have always been welcoming of design work, so this scenario usually  
+ends well. Still, it exposes exactly how the value system (trust and reputation) that drives a lot of  
+open source work fails people who do non-development work (not only design) in this field.  Personally,  
+I’m grateful that people tend to trust my word when I say I’m a designer, or that there have always  
+been developers in the communities I’ve worked with willing to vouch for me if needed. However,  
+it still is extra time and work siphoning energy from the design, and continues a greater trend of  
+lack of trackable credit for design work.
+
+## Coordination and Product Direction
+
+**TG:** None of this is to say that design contributions are not welcome in open-source, in fact  
+I’ve personally found the opposite to be true. But at its core, open-source is, and has generally  
+been, focussed on generating code. This focus is a choice, and this choice has impacts on the  
+community, governance, direction, and more. Given the constraints that go into a decentralized  
+team, often working on different priorities, it can be difficult to know in what direction the  
+project is heading. This process often generates useful software that’s technically sound, after  
+all, as Vincent Van Gogh said, “Great things are done by a series of small things being brought  
+together.” Open-source does a wonderful job of improving large complex software systems by iterating  
+on bite-sized pieces, but this work often comes in spurts when maintainers have time and inspiration.
+
+Great design, on the other hand, is often the product of a well-defined vision. Steve Jobs was  
+famous for saying, “Design is not what it looks like and feels like, it’s how it works.” But we  
+don’t often have the luxury, as designers, of designing how it works, we’re asked to pick up  
+wherever the project is, with no guarantee of impact in the area we are working.
+
+It can be hard to invest into a thorough design process if you’re not sure what’s going to be developed  
+next, or ever. And if you do take that chance and invest in a design, there’s an uncomfortably  
+high chance you’re missing the mark of what the maintainer thinks the next big steps of their project  
+are (possibly) documented only in their mind). If members of a project don’t intentionally take time  
```

Comment:
I think there's something missing here? Maybe 

`(those steps are possibly documented only in their mind)`

Reply:
I believe it should be those steps are (possibly) documented only in their mind. 

I wish to imply that they may not even be documented there.